docs/documentation.md

```diff
@@ -4,220 +4,135 @@ coremltools API Documentation
 
-### Updating the API reference
-
-To create a new version or update the HTML files for the API reference, follow these general steps:
-
-1. Fork and clone the repo and edit the content as usual.
-2. Generate the HTML files by running this script from the root folder: `zsh -i scripts/build_docs.sh`. The HTML files appear in the `docs/_build/html` folder.
-3. Copy the `html` folder to a temporary location, and use it to preview your changes. Repeat 1-3 until finished.
-4. Push the changes to your fork, and create a pull request (PR) to merge them with the `main` branch of Apple's repo.
-5. Checkout the `gh-pages` branch from upstream.
-6. Copy the contents of the `html` folder to the root folder, overwriting the existing contents.
-7. Commit and push your changes to *your* fork, and submit a PR to merge them with the remote `gh-pages` branch.
-
-At this point you can share the following URL to preview the new version, with your GitHub ID for `sample-user`: `https://sample-user.github.io/coremltools/`.
-
 
-#### Fork and clone
+## How the API Reference is auto-generated
+​
+To automatically generate the content for the [API Reference](https://apple.github.io/coremltools/index.html), the Core ML Tools team uses the following process with [Sphinx](https://www.sphinx-doc.org) and [numpydoc](https://pypi.org/project/numpydoc/) to generate the HTML files from the _docstrings_ in the source code:
 
-The following are the specific steps in git corresponding to the first part of general step 1:
-
-1. With your GitHub ID for `sample-user`, fork and clone the `coremltools` repo, change to your fork clone's root folder (`coremltools/`), and add the upstream repo to the list of remotes:
-
-	```shell
-	# Clone your fork to your local machine.
-	git clone git@github.com:sample-user/coremltools.git
-	cd coremltools
-	# Add 'upstream' repo to list of remotes.
-	git remote add upstream https://github.com/apple/coremltools.git
-	# Verify the new remote named 'upstream'
-	git remote -v
-	```
-	
-    You should see the following, with your GitHub ID for `sample-user`:
-    
-    ```shell
-    origin   git@github.com:sample-user/coremltools.git (fetch)
-	origin   git@github.com:sample-user/coremltools.git (push)
-	upstream      https://github.com/apple/coremltools.git (fetch)
-	upstream      https://github.com/apple/coremltools.git (push)
-	```
+1. Set up the Sphinx configuration, and create and edit the `index.rst` file in the `coremltools/docs/` folder, and the other `.rst` files in the `coremltools/docs/source` folder, to establish the documentation's layout and navigation. 
+   
+   This step is performed once (and has already been performed by the team). You don't need to do it again unless you need to make changes to the Sphinx configuration, layout, or navigation, such as adding a new module or MIL op. For details on how to create and edit these files, see [Generating the top-level RST files](#generating-the-top-level-rst-files).
```

Comment:
> such as adding a new module or MIL op

For just adding a MIL op, we don't need to regenerate the rst files. Its only when we want to add a new _categorization_ of MIL ops

Reply:
To make this simpler, I shortened it to "such as adding a new module."